dist/_include-media.scss

```diff
@@ -94,6 +94,89 @@ $unit-intervals: (
   'rem': 0.1
 ) !default;
 
+///
+/// Defines whether support for media queries is available, useful for creating separate stylesheets
+/// for browsers that don't support media queries
+///
+/// @example scss - Disables support for media queries
+///  $im-media-support: false;
+///   @include media(">=tablet") {
```

Comment:
All the `@example` code snippets are using double quotes throughout the code. I'll change them all, then.